lib/tasks/scrapeMovies.rake

```diff
@@ -0,0 +1,75 @@
+namespace :scrapeMovies do 
+  require "net/http"
+  require "uri"
+
+
+  desc "scrape oscars.org for list of best movies"
+  task :movies => :environment do |t, args|
+
+    require 'open-uri'
+    require 'nokogiri'
+
+    #  can call like this if you want
+    # rake scrapeMovies:movies year="2000"
+    year = ENV['year']
```

Comment:
I was testing with individual years ie rake scrapeMovies:movies year="2000"
This is handy code I might want to refer to again - I didn't want to delete it :/